src/Microsoft.Restier.Core/packages.config

```diff
@@ -1,6 +1,18 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Extensions.DependencyInjection" version="1.0.0-rc1-final" targetFramework="net451" requireReinstallation="true" />
-  <package id="Microsoft.Extensions.DependencyInjection.Abstractions" version="1.0.0-rc1-final" targetFramework="net451" requireReinstallation="true" />
+  <package id="Microsoft.Extensions.DependencyInjection" version="1.0.0-rc2-final" targetFramework="net45" />
+  <package id="Microsoft.Extensions.DependencyInjection.Abstractions" version="1.0.0-rc2-final" targetFramework="net45" />
   <package id="Microsoft.OData.Edm" version="6.15.0" targetFramework="net45" />
+  <package id="System.Collections" version="4.0.11-rc2-24027" targetFramework="net45" />
+  <package id="System.Collections.Concurrent" version="4.0.12-rc2-24027" targetFramework="net45" />
+  <package id="System.ComponentModel" version="4.0.1-rc2-24027" targetFramework="net45" />
+  <package id="System.Diagnostics.Debug" version="4.0.11-rc2-24027" targetFramework="net45" />
+  <package id="System.Globalization" version="4.0.11-rc2-24027" targetFramework="net45" />
+  <package id="System.Linq" version="4.1.0-rc2-24027" targetFramework="net45" />
+  <package id="System.Linq.Expressions" version="4.0.11-rc2-24027" targetFramework="net45" />
+  <package id="System.Reflection" version="4.1.0-rc2-24027" targetFramework="net45" />
+  <package id="System.Resources.ResourceManager" version="4.0.1-rc2-24027" targetFramework="net45" />
+  <package id="System.Runtime.Extensions" version="4.1.0-rc2-24027" targetFramework="net45" />
+  <package id="System.Threading" version="4.0.11-rc2-24027" targetFramework="net45" />
+  <package id="System.Threading.Tasks" version="4.0.11-rc2-24027" targetFramework="net45" />
```

Comment:
What are these packages for? If there are not related DI or EF7, I prefer not include RC version now.